src/core/Akka.TestKit/TestKitBase_Receive.cs

```diff
@@ -329,14 +329,14 @@ public async ValueTask<object> PeekOneAsync(TimeSpan? max = null, CancellationTo
         public object PeekOne(CancellationToken cancellationToken)
         {
             return PeekOneAsync(cancellationToken)
-                .ConfigureAwait(false).GetAwaiter().GetResult();
+                .Result;
```

Comment:
We still need to use GetAwaiter().GetResult() because this automatically unwraps all AggregateException, using .Result here breaks behavior consistency.